src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    protected String task_info;
+    protected boolean task_completion;
+
+    public Task(String task_info) {
```

Comment:
It might be a bit better to make the constructor of Task protected in my opinion, since as the super class of Event, Todo and Deadline, it should not be allowed to be instantiated directly.